packages/specification/schemas/resources/UpdatePrescriptionStatusTask.yaml

```diff
@@ -1,5 +1,5 @@
 type: object
-required: [id, resourceType, basedOn, status, businessStatus, intent, focus, for, lastModified, owner]
```

Comment:
@jonathanwelch1-nhs would this change mean the input field would show up as mandatory in the OAS? If so, this should be removed as the input field is optional